.github/workflows/pull-request.yaml

```
@@ -38,21 +38,21 @@ jobs:
           path: flyway-${{ env.FLYWAY_VERSION }}
           key: ${{ runner.os }}-build-flyway-${{ env.FLYWAY_VERSION }}
       - run: "[[ -d flyway-${FLYWAY_VERSION} ]] || (curl -L https://repo1.maven.org/maven2/org/flywaydb/flyway-commandline/${FLYWAY_VERSION}/flyway-commandline-${FLYWAY_VERSION}.tar.gz -o flyway-commandline-${FLYWAY_VERSION}.tar.gz && tar -xzf flyway-commandline-${FLYWAY_VERSION}.tar.gz && rm flyway-commandline-${FLYWAY_VERSION}.tar.gz)"
+      - run: cargo build --locked --all-features --tests
       - run: cargo test --locked --all-features
       - run: |
           sudo systemctl start postgresql.service
           sudo -u postgres createuser $USER
           sudo -u postgres createdb $USER
           psql -c "ALTER USER $USER PASSWORD '$FLYWAY_PASSWORD';"
-          flyway-${FLYWAY_VERSION}/flyway -url="jdbc:postgresql:///" -user=$USER -locations="filesystem:database/sql/" migrate 
-          # Postgres tests should not run in parallel because they use the same database.
-          cargo test --locked --workspace --all-features --jobs 1 postgres -- --ignored --test-threads 1
+          flyway-${FLYWAY_VERSION}/flyway -url="jdbc:postgresql:///" -user=$USER -locations="filesystem:database/sql/" migrate
+        # Postgres tests should not run in parallel because they use the same database.
+      - run: cargo test --locked --all-features --jobs 1 postgres -- --ignored --test-threads 1
```

```suggestion
      - run: cargo test --locked --all-features postgres -- --ignored --test-threads 1
```

Why reduce the number of jobs for building the tests? AFAIK `--jobs` does not affect the number of threads for executing tests.

---

I didn't add this in this PR but yeah I think you're right that it isn't needed. Will apply the suggestion.

---

Ah right - didn't notice it was from before.